api-report/odsp-driver-definitions.api.md

```diff
@@ -216,6 +216,39 @@ export enum OdspErrorType {
     snapshotTooBig = "snapshotTooBig"
 }
 
+// @public
+export const OdspErrorTypes: {
+    readonly outOfStorageError: "outOfStorageError";
+    readonly invalidFileNameError: "invalidFileNameError";
+    readonly snapshotTooBig: "snapshotTooBig";
+    readonly fetchTimeout: "fetchTimeout";
+    readonly fluidNotEnabled: "fluidNotEnabled";
+    readonly cannotCatchUp: "cannotCatchUp";
+    readonly serviceReadOnly: "serviceReadOnly";
+    readonly genericNetworkError: "genericNetworkError";
+    readonly authorizationError: "authorizationError";
+    readonly fileNotFoundOrAccessDeniedError: "fileNotFoundOrAccessDeniedError";
+    readonly offlineError: "offlineError";
+    readonly unsupportedClientProtocolVersion: "unsupportedClientProtocolVersion";
+    readonly writeError: "writeError";
+    readonly fetchFailure: "fetchFailure";
+    readonly fetchTokenError: "fetchTokenError";
+    readonly incorrectServerResponse: "incorrectServerResponse";
+    readonly fileOverwrittenInStorage: "fileOverwrittenInStorage";
+    readonly deltaStreamConnectionForbidden: "deltaStreamConnectionForbidden";
+    readonly locationRedirection: "locationRedirection"; /**
+    * SPO admin toggle: fluid service is not enabled.
+    */
```

Comment:
How did that comment sneak up here? 😆

Reply:
I have no idea. This seems to be an api-extractor bug. It repros consistently between multiple enlistments and CI. Interestingly, there was already a comment in the doc, just in a different place. Fortunately, such content should be ignored / not get propagated anywhere else.